$NetBSD: patch-gemspec,v 1.1 2012/08/12 16:13:03 imil Exp $

OVERRIDE_GEMSPEC is supposed to handle this, but the homepage field is
broken upstream.

--- ../stompserver-0.9.9.gemspec.orig   2012-08-12 12:23:16.000000000 +0000
+++ ../stompserver-0.9.9.gemspec
@@ -104,7 +104,7 @@ files:
 - test/test_stomp_frame.rb
 - test/test_topic_manager.rb
 - test_todo/test_stomp_server.rb
-homepage: ! '    by Patrick Hurley, Lionel Bouton'
+homepage: http://stompserver.rubyforge.org/
 licenses: 
 post_install_message: 
 rdoc_options:
